e2e/helpers/generated_order_helper.js

```diff
@@ -104,8 +116,10 @@ module.exports = {
     if (order.type === 'Blank order (C21)') {
       I.seeAnswerInTab(2, orderHeading, 'Order title', order.title);
       I.seeAnswerInTab(4, orderHeading, 'Order document', order.document);
+      I.seeAnswerInTab(5, orderHeading, 'Date of issue', dateFormat(new Date(), 'd mmmm yyyy'));
```

Comment:
Slightly concerned that you are just making a new date here. Would it be better for the date to be generated when you make the input and then pass it here for assertion?

Reply:
yes will generate before createOrder()